components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -307,16 +351,30 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
                 handleDelete={this.handleDelete}
                 handleAdd={this.addValue}
                 handleSubmit={this.handleSubmit}
-                maxValues={MAX_SELECTABLE_VALUES}
+                handleCancel={closeMembersInviteModal}
                 buttonSubmitText={buttonSubmitText}
                 buttonSubmitLoadingText={buttonSubmitLoadingText}
                 saving={this.state.saving}
                 loading={this.state.loadingUsers}
+                numRemainingText={false}
```

Comment:
you are right, it only needed a modification in the logic for showing/hiding the text. It was just validating for not to be a null value but not for undefined so that's why even when we were not sending the maxValue it was still showing the help text message.